exercise-5.js

```diff
@@ -0,0 +1,35 @@
+function crypto_calculator(money, bitcoin_price, ethereum_price, litecoin_price) {
+    if (money <= 0) {
+        console.log("Insufficient funds");
+        return [0, 0, 0];
+    }
+
+    let bitcoin_max, ethereum_max, litecoin_max = 0;
```

Comment:
Good job on the consolidating similar value variables I hadn't thought about that, but by the consolidating I meant removing these lines entirely.